back/spec/models/custom_field_spec.rb

```diff
@@ -386,6 +386,14 @@ def visit_ranking(_field)
       # 'somewhere else' option should be the last.
       expect(domicile_field.options.last.area).to be_nil
     end
+
+    describe 'ordered_options' do
+      it 'returns the ordered domicile options with keys and titles from areas' do
+        options = domicile_field.ordered_options
+        expect(options.pluck(:key)).to eq(Area.pluck(:id) + ['outside'])
```

Comment:
Just checking, will `Area.pluck(:id)` always return the areas in the same consistent order?

Reply:
Good point. Need to add the ordering in here